components/webui/scripts/start.py

```diff
@@ -10,4 +10,5 @@
 import utils  # NOQA
 
 
-utils.systemd.start('cloudify-webui')
+ctx.logger.info('Starting WebUI Service...')
+utils.start_service_and_archive_properties('cloudify-webui')
```

Comment:
Just a reminder that once we normalize the service's name, we should also change all start/stop scripts to use those names